incubator/kube-aws-ingress-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "kube-aws-ingress-controller.name" . }}
+  namespace: kube-system
+  labels:
+    application: {{ template "kube-aws-ingress-controller.name" . }}
+    component: ingress
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: {{ template "kube-aws-ingress-controller.name" . }}
+      component: ingress
+  template:
+    metadata:
+      labels:
+        application: {{ template "kube-aws-ingress-controller.name" . }}
+        component: ingress
+    spec:
+      {{- if .Values.rbac.create }}
+      serviceAccountName: {{ .Values.rbac.serviceAccountName }}
+      {{- end }}
+      containers:
```

Comment:
Consistently indent lists throughout the chart. Either zero or two spaces. Don't mix styles.

Reply:
I don't get what is the problem here sorry.